README.md

```diff
@@ -6,35 +6,43 @@ This repository applies [BERT](https://github.com/google-research/bert) to named
+
+To create the environment with `conda`, all you need to do is navigate to the `Multilingual_NER` directory and run the following in the command line:
+
+`conda env create -f environment.yml`
+
+You can verify that the required packages installed correctly by activating the environment (`conda activate m_ner`) and running `conda list`.
+
+Partial list of dependencies:  
+
+- pytorch-transformers 1.1.0+, torch 1.2.0+
+- spaCy (2.1.0+)
+- TensorFlow 1.14.0+, keras 2.2.4+ (only used for utility functions - will try to eliminate this dependency down the line)
+- numpy 1.15.4+, pandas 0.25.0+
+- pre-commit 1.18.0+
+- streamlit 0.4.0+
+- (Deployment dependencies - TBD)
+- Miscellaneous minor packages: pathlib, seqeval, tqdm, yaml
 
 ## Usage
 
 `python main.py` to train an English model and save per-epoch checkpoints.
 Add `-lang ru` to train a Russian model.
 
-Once you have a usable model checkpoint, open `python/demo.py` and set the path to the 
-model checkpoint you'd like to use and the demo's default language. Then run `streamlit run demo.py`, which opens a web app you can use to test the model's predictions interactively.
-
-## Requirements
-
--   **NOTE**: Since there are quite a few requirements, we recommend creating a virtual environment when using this package. (use `requirements.txt` when creating environment from scratch)
--   pytorch-transformers 1.1.0+, torch 1.2.0+
--   spaCy (2.1.0+)
--   TensorFlow 1.14.0+, keras 2.2.4+ (only used for utility functions - will try to eliminate this dependency down the line)
--   numpy 1.15.4+, pandas 0.25.0+
--   pre-commit 1.18.0+
--   streamlit 0.4.0+
--   (Deployment dependencies - TBD)
--   Miscellaneous minor packages: pathlib, seqeval, tqdm, yaml
+Once you have a usable model checkpoint, open `python/demo.py` and set the path to the model checkpoint you'd like to use and the demo's default language. Then run `streamlit run demo.py en`, which opens a web app you can use to test the model's predictions interactively. For the Russian demo, change `en` to `ru`.
 
 ## Models
 
--   The English model is a fine-tuned implementation of Google's `bert-base-cased` model, ensembled with spaCy's `en_core_web_lg`, which uses a CNN architecture.
--   The Russian model is a fine-tuned implementation of Google's `bert-base-multilingual-cased` model, ensembled with spaCy's multilingual `xx_ent_wiki_sm` NER model, which uses a CNN architecture.
+- The English model is a fine-tuned implementation of Google's `bert-base-cased` model, ensembled with spaCy's `en_core_web_lg`, which uses a CNN architecture.
+- The Russian model is a fine-tuned implementation of Google's `bert-base-multilingual-cased` model, ensembled with spaCy's multilingual `xx_ent_wiki_sm` NER model, which uses a CNN architecture.
+- Before running the demo or either of the spaCy evaluation notebooks, be sure to run `python -m spacy download MODEL_NAME`.
```

Comment:
I see the instructions here for the spacy download; these should go higher up, before the demo running instructions